apps/server/src/modules/files-storage/service/files-storage.service.ts

```diff
@@ -245,9 +245,10 @@ export class FilesStorageService {
 		}
 	}
 
-	public async downloadFile(schoolId: EntityId, fileRecordId: EntityId, bytesRange?: string): Promise<IGetFile> {
-		const pathToFile = createPath(schoolId, fileRecordId);
-		const response = await this.storageClient.get(pathToFile, bytesRange);
+	public async downloadFile(fileRecord: FileRecord, bytesRange?: string): Promise<GetFileResponse> {
```

Comment:
Die beiden Methoden `downloadFile` und `download` machen doch jetzt das gleiche. Du könntest `download` einfach rausschmeißen und die bisherigen Aufrufe auf `downloadFile` ändern.

Reply:
Nein, werden für verschiedene use cases benutzt

Reply:
Und ich mache kein Refactoring in dem PR! Wobei mir schon paar stellen aufgefallen die ich gerne refactorn würde

Reply:
Naja, die Signatur zu ändern, ist eigentlich auch ein Refactoring, halt nur ein kleineres... aber okay...